src/RenderBoard.spec.ts

```diff
@@ -0,0 +1,22 @@
+import expect from "expect";
+import { renderBoard } from './RenderBoard';
+
+describe("RenderBoard test", () => {
+  it("should return a string that graphically represents a board", () => {
+    const board = [[1, 2], [0, 3]];
+    const graphicalBoard = renderBoard(board);
```

Comment:
renderBoard is the function that you test, it must be in the "When" section. (following the Given/When/Then test pattern)